<?xml version="1.0"?>
<launch>

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find wheel_controller)/config/ros_controllers.yaml" command="load"/>

  <!-- start this controller -->
  <node name="wheel_controller_spawner" pkg="controller_manager" type="controller_manager" args="spawn swerve_controller wheel_controller joint_state_controller" respawn="false" output="screen"/>

</launch>
